src/app/component/shared/components/form-base/form-base.component.ts

```diff
@@ -60,15 +59,8 @@ export class FormBaseComponent implements ComponentCanDeactivate {
   public checkChanges(): boolean {
     const body = this.getFormValues();
     for (const key of Object.keys(body)) {
-      if (Array.isArray(body[key])) {
-        if (body[key].some((item, index) => item !== this.initialValues[key][index])) {
-          return true;
-        }
-      } else {
-        if (body[key] !== this.initialValues[key]
-          && this.formEditProf.nativeElement.classList.contains('ng-touched')) {
-          return true;
-        }
```

Comment:
if you delete this code, you will broke another functionality:

https://user-images.githubusercontent.com/70901435/114194923-a63b2d80-9958-11eb-8dc9-31c9cd0b1df4.MOV

Reply:
https://user-images.githubusercontent.com/46484914/114263994-08e00800-99f1-11eb-94de-c90d5877912b.MOV

Reply:
problem is in another case. please, contact with me for additional info